### ISSUE TITLE
Dynamically Reloading Individual Apps

### ISSUE THREAD

**Issue by frink** [meta-word/RingSocket#5]
Restarting one App without restarting the whole host would be helpful if the apps used are all microservices of sorts. Essentially this allows each .so to be its own microservice which can be deployed separately.

You may already have this in your greater plan. But I didn't see it in the documentation.

**Comment by wbudd** [comment]
Yes, that would absolutely be a nice ability to have! I'm going to need a while to think about if/how I can make that a reality. Definitely not going to be *easy* to implement, but I'll keep this in mind.
